nav2_map_server/src/occ_grid_loader.cpp

```diff
@@ -183,10 +183,6 @@ nav2_util::CallbackReturn OccGridLoader::on_activate(const rclcpp_lifecycle::Sta
   occ_pub_->on_activate();
   occ_pub_->publish(*msg_);
 
```

Comment:
We may not want to completely delete this functionality. We can have a parameter to allow the user to publish the map just once or periodically.